package.json

```diff
@@ -54,6 +54,7 @@
   "dependencies": {
     "bluebird": "^3.4.1",
     "cli-table": "^0.3.1",
+    "deasync": "^0.1.12",
```

Comment:
I'm still learning the whole sync/async thing with node, but this *feels* like a different direction than the typical promisified direction we're taking in the rest of the code. I know that a lot of the first scripts I wrote weren't fully promisified. @pirog what do you think about that?